table_printer.go

```diff
@@ -10,28 +10,36 @@ import (
 
 // DefaultTable contains standards, which can be used to print a TablePrinter.
 var DefaultTable = TablePrinter{
-	Style:          &ThemeDefault.TableStyle,
-	HeaderStyle:    &ThemeDefault.TableHeaderStyle,
-	Separator:      " | ",
-	SeparatorStyle: &ThemeDefault.TableSeparatorStyle,
-	LeftAlignment:  true,
-	RightAlignment: false,
+	Style:                   &ThemeDefault.TableStyle,
+	HeaderStyle:             &ThemeDefault.TableHeaderStyle,
+	HeaderRowSeparator:      "=",
```

Comment:
Please leave the default values empty, so that existing tables won't change with an update of pterm.
```suggestion
	HeaderRowSeparator:      "",
```